content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
```

Comment:
This is a bug, not a new feature. That being said, I understand why we want to have this listed as a "new feature." How about we treat the bug as "We incorrectly allowed GitHub Enterprise as an option when it wasn't supported yet." Something like that. 

```suggestion
* Admin Console now supports GitHub Enterprise as a GitOps repository. 
```